pkgs/top-level/all-packages.nix

```diff
@@ -7041,6 +7041,8 @@ with pkgs;
       protobuf = pkgs.protobuf3_21.overrideDerivation (_: { stdenv = clangStdenv; });
     };
 
+    openbangla-keyboard = libsForQt5.callPackage ../applications/misc/openbangla-keyboard { withIbusSupport = true; };
+
```

Comment:
This should be removed.

Reply:
why?

Reply:
Because you both added a alias throwing this and add this to all-packages.

Unless I understood this wrong.

Reply:
https://github.com/NixOS/nixpkgs/pull/258314/files#diff-29ed0f5fda84c91253503c1664d60a194a324e29518d472adf846f30b8db52e3R598

Reply:
This is in the `ibus-engines` scope, i.e.
I removed `openbangla-keyboard` from top-level, added throws alias for that, and added two new packages (`ibus-engines.openbangla-keyboard` and `fcitx5-openbangla-keyboard`).

I see there's a discrepancy with how ibus and fcitx5 handle their packages, but I didn't want to create a pattern that's never been seen before in nixpkgs.

Reply:
This attribute is declared in the `ibus-engines` attribute set. It is not a top level attribute (see line 7024. https://github.com/NixOS/nixpkgs/pull/258314/files#diff-ab5748dc9567516fefba8344056b51ec1866adeace380f46e58a7af3d619ea22R7024-R7045).

Reply:
Oh ok I see, sorry, this is in the `ibus-engines` sub attribute (damn GitHub with its small diffs size).

You are correct here.